src/main.controller.js

```diff
@@ -80,6 +97,63 @@ plugin.controller('wgnConfigCtrl', ['$scope', '$q', '$routeParams', 'znData', 'z
 			});
 		};
 
+
+		/**
+		 * Selects the configuration to copy.
+		 */
+		$scope.onSelectedConfig = function (config) {
+			if(previousSelectedConfig && Object.keys(previousSelectedConfig).length === 0){
+				config.selected = true;
+				selectedConfig = config;
+			} else {
+				if(previousSelectedConfig.name === config.name){
+					if(previousSelectedConfig.selected === true) {
```

Comment:
unneeded === true. if the value of previousSelectedConfig.selected is true, than will be a truthy value

Reply:
i would personally go with a ternarry operator approach for either config.selected and slecetedConfig. it seems like too many nested if else statements